bluebottle/funding/transitions.py

```diff
@@ -222,7 +241,7 @@ class values(DjangoChoices):
         started = ChoiceItem('started', _('started'))
         succeeded = ChoiceItem('succeeded', _('succeeded'))
         failed = ChoiceItem('failed', _('failed'))
-        cancelled = ChoiceItem('cancelled', _('cancelled'))
+        cancelled = ChoiceItem('activity-refunded', _('cancelled'))
```

Comment:
For consistency use `activity_refunded` instead of `activity-refunded`?
Also refactor `cancelled` and `cancel()` into `activity_refunded`?

Reply:
Nice catch. That should not have changed at all.